src/site/topic20.rst

```diff
@@ -23,6 +23,27 @@ Linear Search
 Iterative
 ---------
 
+* Below is a generic Java implementation of a linear search on an array of type ``T``
+* Other than being a generic method, this should be something you are familiar with
+
+.. code-block:: java
+    :linenos:
+
+    public static <T> int iterativeLinearSearch(T toFind, T[] data) {
```

Comment:
Technically a little broken, see [Arrays#indexOf](https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/util/Arrays.java#L4175-L4188). It's fine to show linear search, but it's not a fully working _Java implementation_.